README.md

```diff
@@ -49,6 +49,53 @@ To lint the JavaScript (report and fix), execute:
+
+Page templates implementing this grid system should use the page grid template. `page-grid.twig` sets up the rows of the top level explicit grid, and handles loading of the lower order grids that directly control content layout with respect to the grid columns (e.g. the content grid `content-grid.twig`).  
+
+In order to preserve the capabilities of seeming to break out of the grid, and of attempting to be future-friendly to sub grids, every lower order grid (i.e. below the level of `page-grid`) must span the `full` width of the grid, and, in addition to anything else the template does, supply css classes to allow its items to span the `full` or `main` width of the grid. For example the `content-grid` provides the css classes `content-grid__item--full` and `content-grid__item--main` respectively.
+
+All nested levels of grid must conform to this `main` / `full` model in order to retain the benefits of this approach.
+
+#### Grid templates
+
+##### `page-grid`
+The top level page grid. It sets up the rows of the top level explicit grid as named areas `start`, `main` and `end`. Typically `start` and `end` would be used to hold the site header and site footer respectively, with everything else located in the `main` row. This is the template to include directly in implementations. Lower level grids, into which the page content actually loads, should be included by `page-grid`.
+
+###### grid areas recommended usage
+- `start`: put the site header here
+- `main`: put everything between the site header and footer here       
+- `end`: put the site footer here
+
+##### `content-grid`
+The grid for all content pages (i.e. not listing pages). In addition to the capability to specify if content spans the entirety of the `main` or `full` sections, this grid defines areas called `primary`, `secondary`, and `menu`. These names are used for both the CSS grid area names and the twig template section names. This grid lays out content with a very similar layout to that of an eLife article.
+
+###### grid areas recommended usage
+- `primary`: the content that makes this page what it is
+- `secondary`: supplementary info, typically used for asides       
+- `menu`: a menu / navigation appropriate to the content level
+   
+
+#### Configuring the grid
+The grid may be configured using the following Sass variables defined in `/src/shared-styles/_grid.scss`:
+
+- `$max-inline-size` the max width of the `main` grid section in pixels (default: `1114px`)
+- `$main-column-count` the number of columns in the `main` area of the grid (default: `12`).
+- `$column-gap` the width between grid columns, also the minimum inline start / end page gutter when CSS grid is not supported by the browser; may be expressed in any css length unit (default: `1.6%`)  
+- `$edge_space-medium`: a medium sized inline start / end page gutter (default: `7vw`), usage controlled by a breakpoint
```

Comment:
For naming consistency this should be the same as the breakpoints as for example wide instead of large etc. Also underscore needs to be replaced with a hyphen.

Example in breakpoints shared styles:
https://github.com/libero/storybook/blob/master/src/shared-styles/_breakpoint.scss#L5